### ISSUE TITLE
Running phpMyAdmin localy on the built in php server

### ISSUE THREAD

**Issue by hel1o-wor1d** [phpmyadmin/phpmyadmin#18705]
Right now phpmyadmin need a webserver to run.
But php has a native server.
So phpmyadmin should run by php self.
I found this ariticle 
https://gist.github.com/iturgeon/0bf432de45a2f19aeb38e2bbae8a4567

**Comment by kamil-tekiela** [comment]
Can you please elaborate? Are you reporting an issue? Suggesting an improvement?

**Comment by Babailan** [comment]
@kamil-tekiela it's improvement since php has it's own webserver

**Comment by MoonE** [comment]
So what should be done on phpmyadmin's side?

Running  phpmyadmin with `php -S` works already, though the built-in webserver is not recommended for production use, see: https://www.php.net/manual/en/features.commandline.webserver.php

**Comment by Babailan** [comment]
This is just complicated i don't really know the ecosystem of php is there a way to run phpmyadmin without installing xampp

**Comment by MoonE** [comment]
You should share a bit more information about your current setup. What operating system, do you already have a database, how did you install php. Is it for production use or development?

Generally,  you  need a webserver to run php, and xampp is probably the easiest way to get going.

**Comment by Babailan** [comment]
I cloned the GitHub repo and already have PHP installed on my system. I'm using the built-in PHP server, as you mentioned. However, it seems to be missing some dependencies that are managed through Composer. Since I don't have much knowledge about using Composer as a package manager, I'm considering switching to MySQL Workbench. I really want to avoid installing XAMPP.


And also this issue is completely unnecessary and someone should close this issue.

**Comment by kamil-tekiela** [comment]
As an aside I don't recommend using XAMPP at all. I used it in the past and I know it's very annoying to setup, use, and keep up-to-date. Either set PHP yourself or use WAMP. Of course that's only suitable for DEV, which I assume what we are talking about here. 

I don't know what issue you would face running with `php -s`. There should be no issues.

**Comment by MoonE** [comment]
You should download the current version or a snapshot from https://www.phpmyadmin.net/downloads if you just want to use phpmyadmin. These already have all the dependencies included.

**Comment by kamil-tekiela** [comment]
Cloning repo is only needed when you want to contribute to the project. If you have no intention of making PRs, just download a release.

**Comment by eyesight-pgs** [comment]
@hel1o-wor1d & @Babailan , please download from official website: https://www.phpmyadmin.net/downloads/
This works perfectly fine with `php -S`. I have tested this one.

**Comment by eyesight-pgs** [comment]
You can also install & run phpMyAdmin using git clone, you just have to follow the documentation (https://docs.phpmyadmin.net/en/latest/setup.html#installing-from-git)

TLDR:
```bash
git clone https://github.com/phpmyadmin/phpmyadmin.git -- depth 1 pma
cd pma
composer update
yarn install
cp config.sample.inc.php config.inc.php
# add 32 byte string to `$cfg['blowfish_secret']`
php -S localhost:80
```

I tried above and is working fine

**Comment by williamdes** [comment]
Thank you @eyesight-pgs 
I am closing this issue, as far as I understand it is solved.
